prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -139,7 +137,12 @@ class ReportFunction(
         val optionsText = request.queryParameters.getOrDefault(OPTION_PARAMETER, "None")
         val httpStatus: HttpStatus =
             try {
-                val options = Options.valueOfOrNone(optionsText)
+                val option = Options.valueOfOrNone(optionsText)
+                if (option.isDeprecated()) {
+                    actionHistory.trackLogs(
+                        ActionLog(InvalidParamMessage("$optionsText is deprecated"), type = ActionLogLevel.warning)
```

Comment:
Add a bit more to the text here.  When I saw the message in the API response I was wondering what kind of error was.  Maybe like "URL parameter <parameter>..."  ... Not sure about the wording, but a bit more to make it obvious what the error is.